pkg/kubelet/apis/config/validation/validation_windows.go

```diff
@@ -20,24 +20,20 @@ limitations under the License.
 package validation
 
 import (
-	"fmt"
-
-	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
 )
 
 // validateKubeletOSConfiguration validates os specific kubelet configuration and returns an error if it is invalid.
-func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
+func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration, warningf func(string, ...interface{})) error {
 	message := "invalid configuration: %v (%v) %v is not supported on Windows"
-	allErrors := []error{}
 
 	if kc.CgroupsPerQOS {
-		allErrors = append(allErrors, fmt.Errorf(message, "CgroupsPerQOS", "--cgroups-per-qos", kc.CgroupsPerQOS))
+		warningf(message, "CgroupsPerQOS", "--cgroups-per-qos", kc.CgroupsPerQOS)
```

Comment:
Would it make sense to explicitly call `klow.Warningf` here instead of passing it down the stack from the `kubelet/app/server.go`?

Reply:
i did it informally like that, but i can change it if the maintainers prefer to call directly.

i think an API package should not import  klog.

Reply:
It makes sense to me. Thanks for the explanations. Let's see what other reviewers say.

not sure if it makes sense to do in this particular case, but to avoid importing klog you can pass context and get the logger from it using `FromContext`.